### ISSUE TITLE
teams webhook missing commit message

### ISSUE THREAD

**Issue by wroghtast** [go-gitea/gitea#8415]
We are using both slack and teams webhooks. In the teams webhook, the commit message is missing.

Regards

**Comment by 6543** [comment]
after #9422 is this still the case?

**Comment by lunny** [comment]
But that haven't been back port to v1.10

**Comment by 6543** [comment]
my suggestion is to move it to 1.11.0 - so we dont need a backport to close this ?

**Comment by 6543** [comment]
@lunny my suggestion is to move it to 1.11.0 - so we dont need a backport to close this ?

**Comment by 6543** [comment]
@wroghtast are you ok to wait for 1.11.0 ?

**Comment by lunny** [comment]
@6543 we could send another PR to fix this issue.

**Comment by lunny** [comment]
Closing as outdated.
